src/Elcodi/Admin/PageBundle/Form/EventListener/PermanentPageSubscriber.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014 Elcodi.com
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ */
+
+namespace Elcodi\Admin\PageBundle\Form\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+/**
+ * Class PermanentPageSubscriber checks if the page that's being handled by the form is persistent to modify the form
+ * fields removing the options that are not present for persistent pages.
+ */
+class PermanentPageSubscriber implements EventSubscriberInterface
```

Comment:
@alch Form listeners are not invented yet XD Form events can only be defined using Subscribers instead of Listeners. Big Kudos for this Symfony implementation (NO)